kem/shortkem/doc.go

```diff
@@ -0,0 +1,25 @@
+// Package shortkem implements a KEM based on short Weierstrass curves.
+package shortkem
+
+import (
+	"crypto"
+	"crypto/elliptic"
+
+	"github.com/cloudflare/circl/ecc/p384"
+	"github.com/cloudflare/circl/kem"
+)
+
+// P256HkdfSha256 is a KEM using P256 curve with HKDF based on SHA-256.
+func P256HkdfSha256() kem.AuthScheme {
+	return short{elliptic.P256(), 0x10, "KemP256HkdfSha256", crypto.SHA256}
```

Comment:
The codepoint shouldn't be hard-coded here. Instead, I would move the HPKE KEA code points from `circl/hpke` into `circl/kem`. (Here and below.)